### PR TITLE
`device-orientation-events`: add intial feature

### PR DESCRIPTION
**Pull request by ddbeck**
This adds device orientation events `DeviceMotionEvent` and `DeviceOrientationEvent`. Despite the somewhat historical notes on https://caniuse.com/deviceorientation, it does seem like there's a usefully interoperable core here.

That said, the MDN docs suggest that the reference frames differ noticeably across platforms. That might be worthy of a note someday.

### CODE REVIEW

File: feature-group-definitions/device-orientation-events.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Device orientation events (intial support)
```
Comment (foolip) [inline feature-group-definitions/device-orientation-events.yml:1]: Can you add a TODO here about the reference frame so that we can come back to it?

Comment (ddbeck) [reply feature-group-definitions/device-orientation-events.yml:1]: Done!

**Review by foolip — APPROVED** [review]

File: feature-group-definitions/device-orientation-events.dist.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+status:
+  baseline: low
+  baseline_low_date: 2023-09-18
+  support:
+    chrome: "31"
+    chrome_android: "31"
+    edge: "12"
+    firefox: "6"
+    firefox_android: "6"
+    safari: "17"
```
Comment (foolip) [inline feature-group-definitions/device-orientation-events.dist.yml:18]: What was the final piece here? Your old markdown tables would be very useful to look at during review :D

Comment (ddbeck) [reply feature-group-definitions/device-orientation-events.dist.yml:18]: All of it: `DeviceMotionEvent`, `DeviceOrientationEvent`, and the window events themselves.

And a lot of that scripting used the precursor to compute-baseline—I'll see if I can quickly reinstate that.

Comment (foolip) [reply feature-group-definitions/device-orientation-events.dist.yml:18]: As an alternative, how about just adding a comment after each BCD key with its Baseline date in a comment?

**Pull request merged**